include/graph.hpp

```diff
@@ -851,7 +851,7 @@ operator<<(std::ostream &os, const port_domain_t &value) {
     return os << static_cast<int>(value);
 }
 
-#ifndef __EMSCRIPTEN__
+#if !defined(__EMSCRIPTEN__) && !(defined(_MSC_VER) && defined(__clang__)) //clang-cl doesn't have source location yet
```

Comment:
The comment only refers to the changed second part of the statement, so it should be either in the commit or specify which part it refers to so that the comment is understandable outside of the diff view.